src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/reduce.cpp

```diff
@@ -19,6 +19,10 @@ std::vector<std::vector<ov::test::InputShape>> inputShapes_dyn = {
     {{{{1, 5}, 19, {1, 5}, {1, 10}}, {{2, 19, 2, 2}, {2, 19, 2, 9}}}},
 };
 
+std::vector<std::vector<ov::test::InputShape>> inputShapes_3D_fuse_dyn = {
+    {{{{1, 5}, 19, {1, 10}}, {{1, 19, 2}, {1, 19, 9}}}},
```

Comment:
```suggestion
    {{{{1, 5}, 19, {1, 10}}, {{1, 19, 2}, {1, 19, 9}, {1, 19, 2}}}},
```
to cover primitive cache

Reply:
Applied! Thanks Chenhu!